examples/pki/config/ca.json

```diff
@@ -1,58 +1,95 @@
 {
-    "root": "examples/pki/secrets/root_ca.crt",
-    "crt": "examples/pki/secrets/intermediate_ca.crt",
-    "key": "examples/pki/secrets/intermediate_ca_key",
-    "password": "password",
-    "address": ":9000",
-    "dnsNames": [
-        "localhost"
-    ],
-    "logger": {
-        "format": "text"
-    },
-    "authority": {
-        "provisioners": [
-            {
-                "name": "mariano@smallstep.com",
-                "type": "jwk",
-                "key": {
-                    "use": "sig",
-                    "kty": "EC",
-                    "kid": "DmAtZt2EhmZr_iTJJ387fr4Md2NbzMXGdXQNW1UWPXk",
-                    "crv": "P-256",
-                    "alg": "ES256",
-                    "x": "jXoO1j4CXxoTC32pNzkVC8l6k2LfP0k5ndhJZmcdVbk",
-                    "y": "c3JDL4GTFxJWHa8EaHdMh4QgwMh64P2_AGWrD0ADXcI"
+	"root": "examples/pki/secrets/root_ca.crt",
+	"federatedRoots": null,
+	"crt": "examples/pki/secrets/intermediate_ca.crt",
+	"key": "examples/pki/secrets/intermediate_ca_key",
+	"address": ":9000",
+	"dnsNames": [
+		"localhost"
+	],
+	"logger": {
+		"format": "text"
+	},
+	"authority": {
+		"provisioners": [
+			{
+				"name": "mariano@smallstep.com",
+				"type": "jwk",
```

Comment:
Move `type` above `name` to be consistent with the format that `step ca init` uses.